report/Report.cpp

```diff
@@ -265,6 +266,17 @@ void InsertSignalBefore(Function *F, Instruction *InsertBefore, int SignalNum) {
       CallInst::Create(Signal, SignalArgs, "", InsertBefore);
 }
 
+/**
+ * @brief Check if the given function is a global var or file
+ * @param fname Function name
+ * @return true if this "function" need to be skipped
+ */
+bool skip(std::string fname) {
```

Comment:
when `#include <iostream>`, there will be some extra functions like
```
__cxx_global_var_init
_GLOBAL__sub_I_example.cpp
```
reporting these functions to a map will cause the floating-pointing exception